src/main/java/de/rexlnico/realtimeplugin/util/Utils.java

```diff
@@ -0,0 +1,77 @@
+package de.rexlnico.realtimeplugin.util;
+
+import de.rexlnico.realtimeplugin.main.Main;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.Random;
+import java.util.logging.Level;
+
+public class Utils {
+
+    public static int overflow(int value, int at) {
+        while (value > at) {
+            value -= at;
+        }
+        return value;
+    }
+
+    public static byte[] httpRequest(String url) {
+        try {
+            return httpRequest(new URL(url));
+        } catch (MalformedURLException e) {
+            Main.getPlugin().getLogger().log(Level.SEVERE, String.format("HTTP request to %s failed", url), e);
+            return null;
+        }
+    }
+
+    public static byte[] httpRequest(URL url) {
+        try {
+            URLConnection conn = url.openConnection();
+            InputStream in = conn.getInputStream();
+            ByteArrayOutputStream bout = new ByteArrayOutputStream();
+
+            byte[] buff = new byte[1024];
+            int len;
+            while ((len = in.read(buff)) != -1) {
+                bout.write(buff, 0, len);
+            }
+
+            bout.close();
+            in.close();
+
+            return bout.toByteArray();
+        } catch (Exception e) {
+            Main.getPlugin().getLogger().log(Level.SEVERE, String.format("HTTP request to %s failed", url), e);
+            return null;
+        }
+    }
+
+    public static Location getRandomLocation(Player player) {
+        Location location;
+        int x;
+        int z;
+        x = new Random().nextInt(player.getLocation().getBlockX() + 25) + 12;
+        z = new Random().nextInt(player.getLocation().getBlockZ() + 25) + 12;
+        if (new Random().nextBoolean()) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [DMI: Random object created and used only once (DMI_RANDOM_USED_ONLY_ONCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dmi-random-object-created-and-used-only-once-dmi-random-used-only-once).
This code creates a java.util.Random object, uses it to generate one random number, and then discards the Random object. This produces mediocre quality random numbers and is inefficient. If possible, rewrite the code so that the Random object is created once and saved, and each time a new random number is required invoke a method on the existing Random object to obtain it.
If it is important that the generated Random numbers not be guessable, you *must* not create a new Random for each random number; the values are too easily guessable. You should strongly consider using a java.security.SecureRandom instead (and avoid allocating a new SecureRandom for each random number needed).

Reply:
You are absolutely right. But in fact this method is not even used :joy: I just placed it there to preserve it (it was previously in the  WeatherWorld class). But I'll change that.

Reply:
Thanks for your reply. Happy to help :)